pkg/channel/distributed/receiver/producer/producer.go

```diff
@@ -21,6 +21,8 @@ import (
 	"errors"
 	"time"
 
+	"knative.dev/eventing-kafka/pkg/channel/distributed/common/testing"
+	"knative.dev/eventing-kafka/pkg/common/client"
 	"knative.dev/eventing-kafka/pkg/common/tracing"
```

Comment:
Why is the `distributed/common/testing` import being added?  Doesn't appear to be used and would be bad to include test code in production if it was.  Assuming it can be removed?

Reply:
I also noticed that. But, it is already used in head's `sarama.go` in here: https://github.com/knative-sandbox/eventing-kafka/blob/master/pkg/channel/distributed/common/kafka/sarama/sarama.go#L37

My refactor is simply moving some stuff out that file to another place. However, I also made the function's scope smaller and made it operate on YAML-string instead of reading stuff from configmap. The configmap reading now is done by the caller and in this case, in this `producer.go` file. That's why the import is added.

IMO, let's tackle that separately, because there's more of this import in files also not changed in this PR.

Reply:
OK, I didn't realize that - bad form on our part and as you suggest we can correct it ourselves - thanks!

Reply:
>  bad form on our part and as you suggest we can correct it ourselves 

@travis-minke-sap sorry, I didn't mean that. I meant, WE can do it together, but not part of this PR ;) Wasn't blaming anybody OR wasn't saying "you fix it"